DEFINITION.md

```diff
@@ -47,3 +47,11 @@ Tecnologias nativas ao cloud empoderam empresas a criarem e rodarem aplicações
 Essas técnicas permitem criar sistemas de baixo acoplamento, resilientes, gerenciáveis e observáveis. Combinadas com automações robustas, elas permitem que os engenheiros façam alterações de alto impacto de forma frequente e previsível, com o mínimo de esforço.
 
 A Cloud Native Computing Foundation procura conduzir a adoção desse paradigma auxiliando e sustentando um ecosistema de projetos de código aberto e não atrelados a nenhum fornecedor. Nós democratizamos padrões estado-da-arte para fazer com que essas inovações sejam acessíveis a todos.
+
+## Français
+
+Les technologies nativement cloud donne le pouvoir aux entreprises de construire et d'exploiter des applications élastiques dans des environnements modernes, dynamiques comme des clouds publics, privés ou bien hybrides. Les conteneurs, les services maillés, les micro services, les infrastructures immuables, les API déclaratives illustrent cette approche.
```

Comment:
I'm not sure that "Services maillés" is more understandable than "solution de service mesh". Sometimes this is better to keep the English term.

Reply:
Hi Smaine,

I agree with you even if on French web site we could have both of them. Technically in plain French it's "Services maillés".